Dockerfile

```diff
@@ -15,4 +15,4 @@ RUN make PREFIX=/go clean binaries
 VOLUME ["/var/lib/registry"]
 EXPOSE 5000
 ENTRYPOINT ["registry"]
-CMD ["cmd/registry/config.yml"]
```

Comment:
Yes, I'll submit a PR to copy dev-config.yml to config.yml at image build to preserve the old location.